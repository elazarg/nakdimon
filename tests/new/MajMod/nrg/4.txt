לְעוּמַּת זֹאת, בְּמַהֲלַךְ הַקַּיִץ מֶזֶג הָאֲוִויר בְּתֵל אָבִיב חַם וְיָבֵשׁ, הלחות גְּבוֹהָה וְלֹא יוֹרְדִים משקעים בִּכְלָל. אֲבָל שִׂימוּ לֵב - אֲחוּזֵי הלחות הַגְּבוֹהִים שֶׁיֵּשׁ בְּתֵל אָבִיב בָּחוֹדָשִׁים אֵלֶּה מְבִיאִים לֹא פַּעַם לעומסי חוֹם גְּבוֹהִים. בְּחוֹדְשֵׁי הַקַּיִץ בְּתֵל אָבִיב, הַטֶּמְפֵּרָטוּרָה הממוצעת בִּשְׁעוֹת הַיּוֹם הִיא כְּ-30 מַעֲלוֹת ובלילה כְּ- 23 מַעֲלוֹת. שתדעו, עַל מֶזֶג הָאֲוִויר בְּתֵל אָבִיב מַשְׁפִּיעָה הַתּוֹפָעָה הנפוצה שֶׁמְּכוּנֶּה אִי הַחוֹם הָעִירוֹנִי. הַגּוֹרֵם הַמֶּרְכָּזִי לַתּוֹפָעָה הַזּוֹ הוּא הַכַּמּוּת הַגְּדוֹלָה שֶׁל כְּלֵי הָרֶכֶב המנועיים שֶׁנּוֹסְעִים בָּרְחוֹבוֹת הָעִיר וּפוֹלְטִים חוֹם.