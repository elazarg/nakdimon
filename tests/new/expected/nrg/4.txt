לְעוּמַּת זֹאת, בְּמַהֲלַךְ הַקַּיִץ מֶזֶג הָאֲוִויר בְּתֵל אָבִיב חַם וְיָבֵשׁ, הַלַּחוּת גְּבוֹהָה וְלֹא יוֹרְדִים מִשְׁקָעִים בִּכְלָל.
אֲבָל שִׂימוּ לֵב - אֲחוּזֵי הַלַּחוּת הַגְּבוֹהִים שֶׁיֵּשׁ בְּתֵל אָבִיב בֶּחוֹדָשִׁים אֵלֶּה מְבִיאִים לֹא פַּעַם לְעוֹמְסֵי חוֹם גְּבוֹהִים.
בְּחוֹדְשֵׁי הַקַּיִץ בְּתֵל אָבִיב, הַטֶּמְפֵּרָטוּרָה הַמְּמוּצַּעַת בִּשְׁעוֹת הַיּוֹם הִיא כְּ-30 מַעֲלוֹת וּבַלַּיְלָה כְּ- 23 מַעֲלוֹת.
שֶׁתֵּדְעוּ, עַל מֶזֶג הָאֲוִויר בְּתֵל אָבִיב מַשְׁפִּיעָה הַתּוֹפָעָה הַנְּפוֹצָה שֶׁמְּכוּנָּה אִי הַחוֹם הָעִירוֹנִי.
הַגּוֹרֵם הַמֶּרְכָּזִי לַתּוֹפָעָה הַזּוֹ הוּא הַכַּמּוּת הַגְּדוֹלָה שֶׁל כְּלִי הָרֶכֶב הַמְּנוֹעִיִּים שֶׁנּוֹסְעִים בִּרְחוֹבוֹת הָעִיר וּפוֹלְטִים חוֹם.